Jenkinsfile

```diff
@@ -73,54 +74,46 @@ pipeline {
                         ''').trim()
                     echo "Uploading uaa-token-lib ${APP_VERSION} build to Artifactory"
                     if (env.BRANCH_NAME == 'master') {
-                        echo 'Branch is master push to MAAXA-MVN, PREDIX-EXT, and maven central'
-                        def uploadSpec = """{
-                            "files": [
-                                    {
-                                        "pattern": "uaa-token-lib-${APP_VERSION}.jar",
-                                        "target": "MAAXA/com/ge/predix/uaa-token-lib/${APP_VERSION}/"
-                                    }
-                                ]
-                            }"""
-
-                        def buildInfo = buildGeArtServer.upload(uploadSpec)
-                        buildGeArtServer.publishBuildInfo(buildInfo)
-
-                        uploadSpec = """{
-                            "files": [
-                                    {
-                                        "pattern": "uaa-token-lib-${APP_VERSION}.jar",
-                                        "target": "PREDIX-EXT/com/ge/predix/uaa-token-lib/${APP_VERSION}/"
-                                    }
-                                ]
-                            }"""
-                        buildInfo = predixExternalArtServer.upload(uploadSpec)
-                        predixExternalArtServer.publishBuildInfo(buildInfo)
+                        ARTIFACTORY_REPO = 'pgog-fss-iam-uaa-mvn'
+                        echo "Branch is master push to ${ARTIFACTORY_REPO}, and maven central"
 
                         sh """#!/usr/bin/env bash
                             set -ex
-                            #Deploy/Release to maven central repository
                             apk update
                             apk add --no-cache gnupg
                             gpg --version
                             ln -s ${WORKSPACE} /working-dir
+
+                            #Deploy/Release to digital grid repository
+                            mvn clean deploy -B -s spring-filters-config/mvn_settings_noproxy.xml \\
+                            -DaltDeploymentRepository=digital-grid.artifactory.repo::default::${ARTIFACTORY_SERVER_URL}/${ARTIFACTORY_REPO} \\
```

Comment:
I like that we are using `mvn deploy` instead of upload spec. Let maven figure out artifactrory path instead of hardcoding it ourselves. Downside is that we are building the artifact again. Small price to pay?

There is no reason to introduce new `digital-grid.artifactory.repo` server. You could reuse existing `artifactory.uaa.releases` and `artifactory.uaa.snapshots`. That way there is no need to change [spring-filters-config](https://github.build.ge.com/predix/spring-filters-config). I tried it with [this build](https://i.ci.build.ge.com/bjmjek1k/ci/blue/organizations/jenkins/Spring%20Filters%2FSpring%20Jwt%20Validation%20Filter/detail/PR-86/22/pipeline) and seems to be working

Reply:
Done, updated server id to `artifactory.uaa.releases` and `artifactory.uaa.snapshots` and closed [spring-filters-config PR](https://github.build.ge.com/predix/spring-filters-config/pull/2).